cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -30,6 +30,140 @@ Cask Data Application Platform Release Notes
+
+- :cask-issue:`CDAP-13982` - Fixed an issue that caused pipelines with too many macros to fail when running in MapReduce.
+
+- :cask-issue:`CDAP-14666` - Fixed an issue with publishing metadata changes for profile assignments.
+
+- :cask-issue:`CDAP-14691` - Fixed a bug that would cause workspace ids to clash when wrangling items of the same name.
+
+- :cask-issue:`CDAP-14702` - Fixed a bug in secure store caused by breaking changes in Java update 171. Users should be able to get secure keys on java 8u171.
+
+- :cask-issue:`CDAP-14708` - Fixed a bug that caused Google Cloud Dataproc clusters to fail provisioning if a firewall rule that denies ingress traffic existed in the project.
+
+- :cask-issue:`CDAP-14709` - Fixed a bug that would cause data preparation to fail when preparing a large file in Google Cloud Storage.
+
+- :cask-issue:`CDAP-14724` - Fixed a bug that caused action only pipelines to fail when running using a cloud profile.
+
+- :cask-issue:`CDAP-14744` - Fixed an issue with adding business tags to an entity.
+
+- :cask-issue:`CDAP-14778` - Fixed an issue in handling metadata search parameters.
+
+- :cask-issue:`CDAP-14779` - Fixed a bug that would cause pipelines to fail on remote clusters if the very first pipeline run was an action only pipeline.
+
+- :cask-issue:`CDAP-14857` - Fixed the standard deviation aggregate functions to work, even if there is only one element in a group.
+
+- :cask-issue:`CDAP-14951` - Fixed a bug in the Google BigQuery sink that would cause pipelines to fail when writing to a dataset in a different region.
+
+- :cask-issue:`CDAP-15001` - Fixed a race condition in processing profile assignments.
+
+- :cask-issue:`CDAP-15013` - Fixed an issue that could cause inconsistencies in metadata.
+
+- :cask-issue:`CDAP-15069` - Fixed an issue with displaying workspace metadata in the UI.
+
+- :cask-issue:`CDAP-15127` - Fixed a race condition in the remote runtime scp implementation that can causing process to hang.
+
+- :cask-issue:`CDAP-15196` - Fixed an issue with metadata search result pagination.
+
+- :cask-issue:`CDAP-15223` - Fixed Wrangler DB connection where a bad JDBC driver can stay in cache for 60 minutes, making DB connection not usable.
+
+- :cask-issue:`CDAP-15249` - Fixed a NullPointerException in Google Cloud Dataproc provision when there is no network being configured.
+
+- :cask-issue:`CDAP-15299` - Fixed a bug that caused some aggregator and joiner keys to be dropped if they hashed to the same value as another key.
+
+- :cask-issue:`CDAP-15332` - Fixed a bug in the RuntimeMonitor that doesn't reconnect through SSH correctly, causing failure in monitoring the correct program state.
+
+- :cask-issue:`CDAP-15369` - Fix Google Cloud Dataproc runtime for Google Cloud Platform projects where OS Login is enabled.
+
+
+Deprecated and Removed Features
+-------------------------------
+
+- :cask-issue:`CDAP-15241` - Deprecated plugins, HDFSMove and HDFSDelete plugins are removed from core plugins.
```

Comment:
Can you clarify this sentence? Not sure I understood the meaning, but if the following is correct, then how about: 
Removed HDFSMove and HDFSDelete plugins from core plugins.
(Wasn't sure if to use "Removed" here or "Deprecated".)